source/specify-a-query.txt

```diff
@@ -0,0 +1,266 @@
+
+   .. code-block:: python
+
+      results = collection.find()
+
+Comparison Operators
+--------------------
+
+Comparison operators analyze the value in a document against the specified value
+in your query filter. The following is a list of common comparison operators:
+
+- Greater than: ``$gt``
+- Less than or Equal: ``$lte``
+- Not equal: ``$ne``
+
+For a full list of comparison operators, see the :manual:`Comparison Query Operators
+</reference/operator/query-comparison/>` guide in the MongoDB Server manual.
+
+The following example specifies a comparison operator in a query filter as a
+parameter to the ``find()`` method. The method finds all documents with a
+``rating`` field value greater than ``2``:
+
+.. io-code-block::
+   :copyable: 
+
+   .. input::
+      :language: python
+
+      results = collection.find({"rating": {"$gt" : 2}})
+
+      for f in results:
+          print(f) 
+
+   .. output::
+
+      {'_id': 1, 'name': 'apples', 'qty': 5, 'rating': 3, 'color': 'red', 'type': ['fuji', 'honeycrisp']}
+      {'_id': 2, 'name': 'bananas', 'qty': 7, 'rating': 4, 'color': 'yellow', 'type': ['cavendish']}
+      {'_id': 4, 'name': 'pineapple', 'qty': 3, 'rating': 5, 'color': 'yellow'}
+
+Logical Operators
+-----------------
+
+Logical operators match documents using logic applied to the results of two or
+more sets of expressions. The following is a list of logical operators: 
+
+- ``$and``, which returns all documents that match the conditions of *both* clauses
+- ``$or``, which returns all documents that match the conditions of *either* clause
+- ``$nor``, which returns all documents that *do not* match the conditions of either clause
+- ``$not``, which returns all documents that *do not* match the expression
+
+For more information about logical operators, see the :manual:`Logical Query Operators
+</reference/operator/query-logical/>` guide in the MongoDB Server manual.
+
+The following example specifies a logical operator in a query filter as a
+parameter to the ``find()`` method. The method finds all documents with a
+``qty`` field value greater than ``5`` **or** a ``color`` field value of
+``"yellow"``:
+
+.. io-code-block::
+   :copyable:
+
+   .. input::
+      :language: python
+
+      results = collection.find({ 
+          "$or": [
+                { "qty": { "$gt": 5 }},
+                { "color": "yellow" }
+          ]
+      })
+
+      for f in results:
+          print(f)
+
+   .. output::
+
+      {'_id': 2, 'name': 'bananas', 'qty': 7, 'rating': 4, 'color': 'yellow', 'type': ['cavendish']}
+      {'_id': 3, 'name': 'oranges', 'qty': 6, 'rating': 2, 'type': ['naval', 'mandarin']}
+      {'_id': 4, 'name': 'pineapple', 'qty': 3, 'rating': 5, 'color': 'yellow'}
+
+Array Operators
+---------------
+
+Array operators match documents based on the value or quantity of elements in an
+array field. The following is a list of array operators:
```

Comment:
Q: is this a complete list?

Reply:
Yes this is all of the array operators. I slightly reworded to make that clear